circle.yml

```diff
@@ -5,8 +5,27 @@ test:
   pre:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
-    - case $CIRCLE_NODE_INDEX in 0) NODE_ENV=test make build-server ;; esac:
-        parallel: true
+    - ? |
+          case $CIRCLE_NODE_INDEX in
+            0) NODE_ENV=test make build-server;
+               ;;
+
+            1) I18N_DIR=$CIRCLE_ARTIFACTS/translate;
+               mkdir -p $I18N_DIR;
+               make translate;
+               mv calypso-strings.pot $I18N_DIR/calypso-strings.pot;
+               if [[ $CIRCLE_BRANCH != "master" ]]; then
+                 git merge-base --fork-point master | xargs git checkout;
```

Comment:
No. That's what we were doing but it introduces inconsistencies when the branch has not been re-based immediately before push.  Master may have new strings since branched with the curl approach.